plugins/router/f5/f5.go

```diff
@@ -712,6 +825,11 @@ func (f5 *f5LTM) Initialize() error {
 		}
 	}
 
+	err = f5.ensurePartitionPathExists(f5.partitionPath)
```

Comment:
I think we can prepend the partition in the URLs.  E.g., in `GetPoolMembers`, `fmt.Sprintf("https://%s/mgmt/tm/ltm/pool/%s/members", f5.host, poolname)` becomes `fmt.Sprintf("https://%s/mgmt/tm/ltm/pool/%s~%s/members", f5.host, poolname, string.Replace(f5.partitionName, "/", "~", -1))`.  However, I do not have an F5 BIG-IP instance handy right now, so I have not tested this.

We will probably want to store the escaped partition name somewhere, or at least use a helper method instead of repeating `string.Replace(...)` everywhere.